java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java

```diff
@@ -184,7 +194,7 @@
     private final Names names;
     private static final Logger LOG = Logger.getLogger(CasualDiff.class.getName());
 
-    private Map<Integer, String> diffInfo = new HashMap<Integer, String>();
+    private Map<Integer, String> diffInfo = new HashMap<>();
```

Comment:
It is ok to remove the types in this constructor, it's easy to understand that "diffInfo" is a map of <Integer, String>